rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2549,6 +2549,44 @@ void voteFederationChange_rollback_returnsSuccessResponseCode() {
 
         }
 
+        @Test
+        void voteFederationChange_addSameFederatorPublicKeyBySameAuthorizerTwice_returnsGenericResponseCodeOnSecondCall() {
```

Comment:
why is it failing? because of the same key or because the same authorizer voted? Is not clear by the test name.
I'd simply put `_addFederatorPublicKeyBySameAuthorizerTwice`, sounds better imho

Reply:
and just `_returnsGenericErrorResponseCode`, i think is implied that it would be on the second call :)

Reply:
Updated.